index.js

```diff
@@ -122,6 +123,24 @@ function handleShell(fn, command, options) {
 	return fn(file, args, options);
 }
 
+function makeAllStream(spawned) {
+	const mixed = mergeStream();
```

Comment:
Maybe this should be initialized after the next three lines, so that it's not initialized if `null` is returned.

Reply:
👍